apps/api/src/common/auth.middleware.ts

```diff
@@ -30,19 +30,19 @@ export class AuthenticationMiddleware implements NestMiddleware {
 
       // Retrieve user roles
       const resourceAccess = user['resource_access'];
-      const ltcvx = resourceAccess && resourceAccess[process.env.AUTH_CLIENTID as any];
-      const roles = ltcvx && ltcvx.roles;
-      res.locals.roles = roles;
-      const applicationUser = await this.employeeService.resolveUser(user.sub, {
+      if (resourceAccess && process.env.AUTH_CLIENTID) {
+        res.locals.roles = resourceAccess[process.env.AUTH_CLIENTID]?.roles;
```

Comment:
I wonder if the roles is set in dev or test because it is undefined in local env.